src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -46,138 +46,104 @@ class CompanyStep extends React.Component {
-            addressCity: ReimbursementAccountUtils.getDefaultStateForField(props, 'addressCity'),
-            addressState: ReimbursementAccountUtils.getDefaultStateForField(props, 'addressState'),
-            addressZipCode: ReimbursementAccountUtils.getDefaultStateForField(props, 'addressZipCode'),
-            companyPhone: ReimbursementAccountUtils.getDefaultStateForField(props, 'companyPhone'),
-            website: ReimbursementAccountUtils.getDefaultStateForField(props, 'website', this.defaultWebsite),
-            companyTaxID: ReimbursementAccountUtils.getDefaultStateForField(props, 'companyTaxID'),
-            incorporationType: ReimbursementAccountUtils.getDefaultStateForField(props, 'incorporationType'),
-            incorporationDate: ReimbursementAccountUtils.getDefaultStateForField(props, 'incorporationDate'),
-            incorporationState: ReimbursementAccountUtils.getDefaultStateForField(props, 'incorporationState'),
-            hasNoConnectionToCannabis: ReimbursementAccountUtils.getDefaultStateForField(props, 'hasNoConnectionToCannabis', false),
-        };
-
-        // These fields need to be filled out in order to submit the form
-        this.requiredFields = [
-            'companyName',
-            'addressStreet',
-            'addressCity',
-            'addressState',
-            'addressZipCode',
-            'website',
-            'companyTaxID',
-            'incorporationDate',
-            'incorporationState',
-            'incorporationType',
-            'companyPhone',
-            'hasNoConnectionToCannabis',
-        ];
-
-        // Map a field to the key of the error's translation
-        this.errorTranslationKeys = {
-            companyName: 'bankAccount.error.companyName',
-            companyPhone: 'bankAccount.error.phoneNumber',
-            website: 'bankAccount.error.website',
-            companyTaxID: 'bankAccount.error.taxID',
-            incorporationDate: 'bankAccount.error.incorporationDate',
-            incorporationType: 'bankAccount.error.companyType',
-            hasNoConnectionToCannabis: 'bankAccount.error.restrictedBusiness',
-            incorporationState: 'bankAccount.error.incorporationState',
-        };
     }
 
-    /**
-     * @param {String} inputKey
-     * @param {String} value
-     */
-    clearErrorAndSetValue(inputKey, value) {
-        const newState = {[inputKey]: value};
-        this.setState(newState);
-        ReimbursementAccount.updateReimbursementAccountDraft(newState);
-        this.clearError(inputKey);
+    componentWillUnmount() {
+        BankAccounts.resetReimbursementAccount();
     }
 
     /**
-     * @returns {Boolean}
+     * @param {Object} values - form input values passed by the Form component
+     * @returns {Object} - object values error strings
      */
-    validate() {
-        const errors = {};
+    validate(values) {
+        const errorTexts = {};
```

Comment:
Let's rename this to errors and update all usages of it. See my comment below about removing the other declared `errors` variable .
```suggestion
        const errors = {};
```

Reply:
done